### DISCUSSION TITLE
strimzi kafka duplicate topic name - not getting deleting

### DISCUSSION THREAD

**Discussion by vjvel** [strimzi/strimzi-kafka-operator#7760] [Q&A]
kafkatopic resource output:

```
preprod.messagestatus                                                                                                                                         strimzi-kafka-cluster-oci-preprod   3            1        
preprod.messagestatus---e0de2736c53e03734b3c1c19166fd5a36488fcd6                                                                                              strimzi-kafka-cluster-oci-preprod   1            1     
```   

I am not able to delete the topic using kafka-topics.sh command also I tried delete using ` kubectl delete kafkatopic ' but its keep on creating with some random name,  I need to create topic with 3 partition and 1 replica but same topic was exist before with 3 replicas with 1 partition.

I tried delete the topic name, its getting generating everytime.

**Comment by scholzj** [comment]
Do you have disabled topic auto-creation? If not, maybe the topic i always immediately auto-created by some client?
